README.md

```diff
@@ -163,12 +165,20 @@ Usage : upjs-plato [options] -d <output_dir> <input files>
       Specify a ESLint configuration file for ESLint linting
   -b, --babelConfig : String
       Specify a Babel configuration file for project parsing
+  -p, --projectRoot : String
+      Root directory of the project to analyze. Needed to run audit/outdated/depngn analysis
 ```
 
-**Example**
+**Examples**
 
 ```shell
 upjs-plato -r -d report src
+# analyze the `src` dir and generates the report in the `report` dir
+```
+
+```shell
+upjs-plato -r -p ./ -T 20 -r report src
+# analyze the `src` dir, also run audit, outdated, and depng with node 20 as the target, store report in the `report` dir
```

Comment:
@arielj Related to my previous comment, I guess it is not clear to me what is the difference between `-p ./` and `src` in this command?